.mailmap

```diff
@@ -1325,3 +1325,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Islem BOUZENIA <fi_bouzenia@esi.dz>
```

Comment:
```suggestion
Islem BOUZENIA <fi_bouzenia@esi.dz> Islem BOUZENIA <61057090+islem-esi@users.noreply.github.com>
```

Reply:
Seems like you had a previous entry if I interpreted the error message correctly. I do not really follow this, but you should have a line similar to the above, with both your email addresses.

Reply:
The message is confusing but the problem is that neither of the names here actually matches what is in the git commits (`islem-esi <fi_bouzenia@esi.dz>`). The line to be added should be:
```
Islem BOUZENIA <fi_bouzenia@esi.dz> islem-esi <fi_bouzenia@esi.dz>
```
There the name "islem-esi" is what is actually in the commits.

After adding that line the `bin/mailmap_check.py` script should be run to sort the file.